plugins/pencilblue/templates/admin/sites/localization.html

```diff
@@ -0,0 +1,211 @@
+^tmp_admin=head^
+<div class="col-md-9">
+    ^tmp_admin=elements=error_success^
+    ^tmp_admin=elements=sub_nav^
+    ^tmp_admin=elements=tab_nav^
+    <style>
+        .form-group textarea {
+            height: 125px;
+        }
+        #localization-plugin-dropdown, #localization-locale-dropdown{
+            width: 100%;
+        }
+    </style>
+
+    <form name="localizationKVForm" ng-submit="saveLocales()" novalidate>
+        <fieldset>
+            <div class="tab-content">
+                <!--Page for Adding Locales-->
```

Comment:
We have a couple of tabs on the localization override view. I'd like to just have one that contains all the relevant information for a person using this, if you have any suggestions about making this view flow better let me know.